db-init/src/main/resources/database/migrations/V4.3__Update_Claim_Table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE claim 
+ADD COLUMN vbms_id VARCHAR,
+ADD COLUMN off_ramp_reason VARCHAR, 
+ADD COLUMN presumptive_flag BOOLEAN, 
+ADD COLUMN disability_action_type VARCHAR,
+ADD COLUMN in_scope BOOLEAN,
```

Comment:
Should be in claim_submission. See https://dsva.slack.com/archives/C01Q7979Z7D/p1675959892218709?thread_ts=1675912916.651659&cid=C01Q7979Z7D

Reply:
Per the referenced slack discussion to clarify this comment
presumptive_flag, and disability_action_type fields should exist in the claim table -- these values shouldn't change if the same claim was resubmitted. 

values for off_ramp_reason and in_scope need to be moved.

Reply:
in_scope moved in V4.8 and V4.11.